docs/Breaking-Changes.md

```diff
@@ -48,4 +48,4 @@ _none_
 
 ## 0.2.5 ##
 
-This wiki was started with v0.2.5.
\ No newline at end of file
+This wiki was started with v0.2.5.
```

Comment:
What change are you making here? Maybe an accidental newline?

Reply:
The vim editor automatically adds a newline symbol at the end of this file. I have removed it.